parts/kubernetesagentcustomdata.yml

```diff
@@ -30,7 +30,10 @@ write_files:
   content: |
     [Service]
     ExecStart=
-    ExecStart=/usr/bin/docker daemon -H fd:// --storage-driver=overlay
```

Comment:
add the comment here as well ?

# Note, the preceeding dash tells systemd to ignore non-zero exit code